noir-projects/noir-contracts/contracts/test/avm_test_contract/src/main.nr

```diff
@@ -163,6 +163,7 @@ pub contract AvmTest {
      * General Opcodes
      ************************************************************************/
     #[external("public")]
+    #[view]
```

Comment:
Needed to do this because where it's called it's called statically and the new API doesn't support calling of non-view function statically because `self.view(...)` expects `PublicStaticCallInterface` on the input and not `PublicCallInterface`.